apps/web/src/store/nft-gallery.ts

```diff
@@ -1,21 +1,39 @@
 import type { Nft } from 'lens';
 import { create } from 'zustand';
 
-interface Item extends Nft {
-  id: string;
+export interface Item extends Nft {
```

Comment:
make the interface name more descriptive if we are exporting to other components too 🙇🏼